libusb/os/windows_winusb.c

```diff
@@ -2708,9 +2708,11 @@ static int winusbx_open(int sub_api, struct libusb_device_handle *dev_handle)
 				default:
 					return LIBUSB_ERROR_IO;
 				}
+			} else if (INVALID_HANDLE_VALUE == file_handle){
```

Comment:
It is a bit confusing that you use Yoda syntax here, while using normal syntax in line 2701.